src/uu/dd/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "uu_dd"
+version = "0.0.7"
+authors = ["uutils developers"]
+license = "MIT"
+description = "dd ~ (uutils) copy and convert files"
+
+homepage = "https://github.com/uutils/coreutils"
+repository = "https://github.com/uutils/coreutils/tree/master/src/uu/dd"
+keywords = ["coreutils", "uutils", "cross-platform", "cli", "utility"]
+categories = ["command-line-utilities"]
+edition = "2018"
+
+[lib]
+path = "src/dd.rs"
+
+[dependencies]
+byte-unit = "4.0"
+clap = { version = "2.33", features = [ "wrap_help" ] }
+debug_print = "1.0"
```

Comment:
You don't use this dependency anymore. Can you remove it?

Reply:
Nice catch. It is removed.